### PR TITLE
Add ability for user to supply default location for Tree Widget

### PR DESCRIPTION
**Pull request by bsteinbk**
Design Review would like the ability to set the default location to the left panel.

### CODE REVIEW

File: packages/itwin/tree-widget/src/components/TreeWidgetUiItemsProvider.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 /* eslint-disable react/display-name */
-import type {
+import {
```
Comment (aruniverse) [inline packages/itwin/tree-widget/src/components/TreeWidgetUiItemsProvider.tsx:6]: i think youll have to run `lint:fix`

File: packages/itwin/tree-widget/src/types.ts [hunk 2]
```diff
@@ -16,31 +17,33 @@ export interface IModelContentTreeProps
...
 
 export type SpatialTreeProps = Omit<
-SpatialContainmentTreeProps,
-"iModel" | "width" | "height"
+  SpatialContainmentTreeProps,
+  "iModel" | "width" | "height"
 >;
 
 export interface TreeWidgetOptions {
+  defaultPanelLocation?: StagePanelLocation,
```
Comment (aruniverse) [inline packages/itwin/tree-widget/src/types.ts:45]: im surprised it took this long for a request like this to come through...

should we instead just expose an obj for users to set their preferred AbstractWidgetProps ?

Comment (bsteinbk) [reply packages/itwin/tree-widget/src/types.ts:45]: These locations are not defined in AbstractWidgetProps, but allowing those to be set may also be a good addition.

Comment (bsteinbk) [reply packages/itwin/tree-widget/src/types.ts]: @aruniverse - since you own this repo I will let you resolve this if you are ok with response.

**Review by aruniverse — APPROVED** [review]

**Pull request merged**